Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -690,6 +691,16 @@ open class NavigationMapView: UIView {
         }
         
         let cameraOptions = CameraOptions(cameraState: mapView.cameraState)
+        
+        if let heading = heading {
+            userCourseView.update(location: location,
+                                  pitch: cameraOptions.pitch!,
+                                  direction: heading.trueHeading,
```

Comment:
If `trueHeading` is negative, fall back to `magneticHeading`, and if that’s also negative, fall back to `course`.